borg/remote.py

```diff
@@ -93,8 +94,11 @@ def serve(self):
                             f = getattr(self.repository, method)
                         res = f(*args)
                     except BaseException as e:
-                        logging.exception('Borg %s: exception in RPC call:', __version__)
-                        logging.error(sysinfo())
+                        if not isinstance(e, (Repository.DoesNotExist, Repository.AlreadyExists, Repository.CheckNeeded,
+                                              IntegrityError, PathNotAllowed, Repository.ObjectNotFound,
+                                              InvalidRPCMethod)):
```

Comment:
No, these are the special-cased exceptions handled [here](https://github.com/enkore/borg/blob/issue/792/borg/remote.py#L234). These are the only Errors that the error-handling code in archiver.py:main will see.